src/components/Footer/index.sass

```diff
@@ -0,0 +1,48 @@
+.contact-box
+    width: 250px
+
+.footer
+    background-color: black
+    color: white
+
```

Comment:
Also sass also we can use { } , right? if so lets use, readabilty will be better

Reply:
in scss we can use {}, but not in saas, let me know i am wrong